spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -18,6 +18,60 @@ describe("kong start/stop #" .. strategy, function()
     helpers.clean_prefix()
   end)
 
+  it("with #referenced values [fails]", function()
+    local _, stderr, stdout = assert(helpers.kong_exec("start", {
+      prefix = helpers.test_conf.prefix,
+      database = helpers.test_conf.database,
+      pg_password = "{vault://env/pg_password}",
+      pg_database = helpers.test_conf.pg_database,
+      cassandra_keyspace = helpers.test_conf.cassandra_keyspace
```

Comment:
If vaults are behind a feature flag, shouldn't there be one here (and in other tests that use vaults)?